Console/Command/Bill.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Paytrail\PaymentService\Console\Command;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class Bill extends \Symfony\Component\Console\Command\Command
+{
+    /** @var \Paytrail\PaymentService\Model\Recurring\Bill */
+    private $bill;
+
+    /** @var \Magento\Framework\App\State */
+    private $state;
+
+    public function __construct(
+        \Paytrail\PaymentService\Model\Recurring\Bill\Proxy $notify,
```

Comment:
https://github.com/magento/magento-coding-standard/issues/18

@bartoszkaluzny-solteq should this be changed?